test/string.cpp

```diff
@@ -1570,55 +1570,55 @@ class string_test
             });
         }
 
-        // insert(size_type, string const&, size_type, size_type)
-        {
-            fail_loop([&](storage_ptr const& sp)
-            {
-                string s(t.v1, sp);
-                s.insert(1, string(t.v2), 1, 3);
-                BOOST_TEST(s == std::string(
-                    t.v1).insert(1, t.s2, 1, 3));
-            });
+        //// insert(size_type, string const&, size_type, size_type)
```

Comment:
need a comment explaining why this is disabled, put your name in caps `// SDK` or something like that so the reader knows it is editorial.

Reply:
Added